lib/cWarnings.mli

```diff
@@ -19,3 +19,5 @@ val set_flags : string -> unit
 (** Cleans up a user provided warnings status string, e.g. removing unknown
     warnings (in which case a warning is emitted) or subsumed warnings . *)
 val normalize_flags_string : string -> string
+
+val pr_warnings : category_name:(string * string option) option -> Pp.std_ppcmds
```

Comment:
Also, following a remark by @ejgallego, printing should not be done here. The warning API should provide a way to get a (filtered) list of warnings and their status. It is then up to the IDE (or coqtop) to print them.